plugin.video.hbogoeu/operators.md

```diff
@@ -0,0 +1,212 @@
+Last update: 2020-04-08
```

Comment:
Is this file required for this add-on submission? Or was it added by accident?

Reply:
Hello, no, it's not required just documentation, forgot to add to the exclude list will remove it now :)